library/runCmds.py

```diff
@@ -9,74 +19,76 @@
 # for expat exceptions...
 import xml
 import ast
-
 import re
-import sys
+try:
+    from thread import get_ident as _get_ident
+except ImportError:
+    from dummy_thread import get_ident as _get_ident
+try:
+    from _abcoll import KeysView, ValuesView, ItemsView
+except ImportError:
+    pass
```

Comment:
Why is the try block eating the ImportError instead of pulling the items from a different location.